automatic/gitextensions/gitextensions.nuspec

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>gitextensions</id>
+    <version>2.51.02</version>
+    <title>Git Extensions</title>
+    <authors>Henk Westhuis</authors>
+    <owners>chocolatey</owners>
+    <licenseUrl>https://github.com/gitextensions/gitextensions/blob/master/LICENSE.md</licenseUrl>
+    <projectUrl>http://gitextensions.github.io/</projectUrl>
+    <iconUrl>https://cdn.rawgit.com/ferventcoder/chocolatey-packages/02c21bebe5abb495a56747cbb9b4b5415c933fc0/icons/gitextensions.svg</iconUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description><![CDATA[## Features
+
+* Windows Explorer integration for Git
+* Feature rich user interface for Git
+]]></description>
+    <summary>Git Extensions is a graphical user interface for Git that allows you to control Git without using the commandline.</summary>
+    <releaseNotes>https://github.com/gitextensions/gitextensions/blob/master/GitUI/Resources/ChangeLog.md</releaseNotes>
+    <tags>foss gitextensions git gui admin</tags>
```

Comment:
missing `cross-platform` tag

Reply:
Will do. But did you known that the tag search is not working? e.g. https://chocolatey.org/packages?q=tag%3Across-platform does not return anything.

Reply:
jah, for now search only with `cross`